setup.cfg

```
@@ -0,0 +1,46 @@
+[metadata]
+name = pytest-arraydiff
+url = https://github.com/astrofrog/pytest-arraydiff
+author = The Astropy Developers
+author_email = astropy.team@gmail.com
+classifiers =
+    Development Status :: 4 - Beta
+    Framework :: Pytest
+    Intended Audience :: Developers
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: Implementation :: CPython
+    Topic :: Software Development :: Testing
+    Topic :: Utilities
+license = BSD
+description = pytest plugin to help with comparing array output from tests
+long_description = file: README.rst
+
+[options]
+zip_safe = False
+packages = find:
+python_requires = >=3.6
+setup_requires =
+    setuptools_scm
+install_requires =
+    pytest>=4.6
+    numpy
+    astropy
+
+[options.entry_points]
+pytest11 =
+    pytest_arraydiff = pytest_arraydiff.plugin
+
+[tool:pytest]
+minversion = 4.6
+testpaths = tests
+xfail_strict = true
```

To avoid the `PytestUnknownMarkWarning`:
```suggestion
xfail_strict = true
markers =
    array_compare : for functions using array comparison
```